doc/flame/inputs/gesture_input.md

```diff
@@ -490,3 +490,44 @@ More information about how to define hitboxes can be found in the hitbox section
 
 An example of how to use it can be seen
 [here](https://github.com/flame-engine/flame/blob/main/examples/lib/stories/input/gesture_hitboxes_example.dart).
+
+
+### MouseMovement on the game level
+
+To give a `Game` class the ability to handle mouse movement/hover events,
+use the `MouseMovementDetector` mixin.  
+
+You can then override an `onMouseMove` method. This method receives
+[`PointerHoverInfo`](https://pub.dev/documentation/flame/latest/events/PointerHoverInfo-class.html)
+which provides the position of the hover event, as well as the raw
+[`PointerHoverEvent`](https://api.flutter.dev/flutter/gestures/PointerHoverEvent-class.html)
+
+```{warning}
+This should not be used on platforms that do not support mouse cursors.
```

Comment:
can we make this warning more clear as to what happens?
is the handler ignored?
or will it error?
that is relevant to multi-platform games